src/runner/PlutoRunner.jl

```diff
@@ -1058,9 +1058,19 @@ function show_richest(io::IO, @nospecialize(x))::Tuple{<:Any,MIME}
     if mime isa MIME"text/plain" && use_tree_viewer_for_struct(x)
         tree_data(x, io), MIME"application/vnd.pluto.tree+object"()
     elseif mime isa MIME"application/vnd.pluto.tree+object"
-        tree_data(x, IOContext(io, :compact => true)), mime
+        try
+            tree_data(x, IOContext(io, :compact => true)), mime
+        catch
+            show(io, MIME"text/plain"(), x)
+            nothing, MIME"text/plain"()
+        end
     elseif mime isa MIME"application/vnd.pluto.table+object"
-        table_data(x, IOContext(io, :compact => true)), mime
+        try
+            table_data(x, IOContext(io, :compact => true)), mime
+        catch
+            show(io, MIME"text/plain"(), x)
+            nothing, MIME"text/plain"()
```

Comment:
```suggestion
            return (nothing, MIME"text/plain"())
```
A general nitpick. I appreciate that there is something beautiful in how Julia figures out the return type even for tuples, but I wonder whether explicit `return` statements are more clear for readers of the code. Especially when there are multiple points at which the function can return